### DISCUSSION TITLE
Github Students

### DISCUSSION THREAD

**Discussion by joshua094** [community/community#29941] [GitHub Education] [answered]
Just got into github student developers plan, great but I can't seem tofind my way yet.

**Comment by TalMalchi** [comment]
Hi Josh, what do you mean to find your way?
There are many helpful things that can help you!

**Comment by divais** [comment, marked as answer]
Hi @joshua094 
Have you checked [Global Campus](https://education.github.com/globalcampus/student). 

GitHub Global Campus is a portal from which you can access all your GitHub Education benefits and resources, all in one place. On the GitHub Global Campus portal, students can:

- Connect with a local Campus Expert. For more information on campus experts, see "[About Campus Experts](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/about-campus-experts)."
- Explore and claim offers for free industry tools from the [Student Developer Pack](https://education.github.com/pack).
- See upcoming in-person and virtual events for students, curated by GitHub Education and student leaders.
- View assignments from [GitHub Classroom](https://classroom.github.com/) with upcoming due dates.
- Stay in the know on what the community is interested in by rewatching recent [Campus TV](https://www.twitch.tv/githubeducation) episodes. Campus TV is created by GitHub and student community leaders and can be watched live or on demand.
- Discover and contribute to student-created repositories from GitHub Community Exchange. For more information, see "[About GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange)."

**Comment by Abhishekshukla1234** [comment]
how to solve the problem does not have a commit checkout. please provide information regarding this issue.

**Comment by Rajat-S-17** [comment]
My college did not provided me with an email address with the domain, I have my college ID card as a proof how can I get student benefit.
